5.3.lahsheshanit.py

```diff
@@ -0,0 +1,29 @@
+# The function extracts the secret messages from the logo file
```

Comment:
I would use documentation instead of comments https://realpython.com/documenting-python-code/#commenting-vs-documenting-code

Reply:
I fixed it.